src/java/Controller/PDF.java

```diff
@@ -48,7 +48,7 @@
     private PDFinfo pdfinfo;
 
 	private String filePath = "C:/Users/syntel/Music/"; //use getfilepath in the email logic?
-    private final static String imagePath = "C:\\trainingkb\\WebSproject\\web\\resources\\img\\pdf_banner.png"; //path to AS logo
+    private final static String imagePath = "C:\\Examples\\WebSproject\\web\\resources\\img\\pdf_banner.png"; //path to AS logo
```

Comment:
Is it just a coincidence that this is the exact path my project is saved in? I didn't think anyone else had an "Examples" directory